test_istgt.sh

```diff
@@ -123,35 +128,26 @@ run_and_verify_iostats() {
 		[[ $? -ne 0 ]] && echo "mount for $device_name" && exit 1
 
 		sudo dd if=/dev/urandom of=/mnt/store/file1 bs=4k count=10000 oflag=direct
-		$ISTGTCONTROL iostats
-		var1="$($ISTGTCONTROL iostats | grep -oP "(?<=TotalWriteBytes\": \")[^ ]+" | cut -d '"' -f 1)"
+		istgtcontrol iostats
+		var1="$(istgtcontrol iostats | grep -oP "(?<=TotalWriteBytes\": \")[^ ]+" | cut -d '"' -f 1)"
 		if [ $var1 -eq 0 ]; then
 			echo "iostats command failed" && exit 1
 		fi
 
 		sudo dd if=/dev/urandom of=/mnt/store/file1 bs=4k count=10000 oflag=direct
-		$ISTGTCONTROL iostats
-		var2="$($ISTGTCONTROL iostats | grep -oP "(?<=TotalWriteBytes\": \")[^ ]+" | cut -d '"' -f 1)"
+		istgtcontrol iostats
+		var2="$(istgtcontrol iostats | grep -oP "(?<=TotalWriteBytes\": \")[^ ]+" | cut -d '"' -f 1)"
 		if [ $var2 -eq 0 ]; then
 			echo "iostats command failed" && exit 1
 		fi
 
 		if [ "$var2" == "$var1" ]; then
 			echo "iostats command failed, both the values are same" && exit 1
-		else
-			echo "iostats test passed"
+		else echo "iostats test passed"
 		fi
 
 		sudo umount /mnt/store
 		logout_of_volume
-		readReqTime="$($ISTGTCONTROL -q iostats | jq '.Replicas[0].ReadReqTime' | cut -d '"' -f 2)"
```

Comment:
here as well, it seems - some code that got checked in got removed.
Please, make sure that you hadn't removed any new code